applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -397,7 +397,10 @@ maybe_resource_to_endpoints(#resrc{id=Id
                             ,OffnetJObj
                             ,Endpoints
                            ) ->
-    CallerIdNumber = wapi_offnet_resource:outbound_caller_id_number(OffnetJObj),
+    CallerIdNumber = case whapps_config:get_is_true(?SS_CONFIG_CAT, <<"cid_rules_honor_diversions">>, 'true') of
+        'false' -> wapi_offnet_resource:outbound_caller_id_number(OffnetJObj);
+        'true' -> check_diversion_fields(OffnetJObj)
+    end,
```

Comment:
Please put the `whapps_config` call in a macro named something like `RULES_HONOR_DIVERSION`.

Also, indent 401-3 properly